frappe/database/query.py

```diff
@@ -0,0 +1,269 @@
+import operator
+from typing import Any, Dict, List, Tuple, Union
+
+import frappe
+from frappe.query_builder import Criterion, Order, Field
+
+class Query:
+	def __init__(self):
+		self.operator_map = {
+			"+": operator.add,
+			"=": operator.eq,
+			"-": operator.sub,
+			"!=": operator.ne,
+			"<": operator.lt,
+			">": operator.gt,
+			"<=": operator.le,
+			">=": operator.ge,
+			"in": self.func_in,
+			"not in": self.func_not_in,
+			"like": self.like,
+			"not like": self.not_like,
+			"regex": self.func_regex,
+			"between": self.func_between
+		}
```

Comment:
The `operator_map` along with the 'custom' (non `operator.*`) operators (`staticmethod`s) don't seem to be instance-specific. Can they be separated out of the `Query` class?

Reply:
Fixed by 4de0cbd